src/briefcase/commands/create.py

```diff
@@ -367,28 +326,27 @@ def _download_support_package(self, app):
                 custom_support_package = True
                 self.logger.info(f"Using custom support package {support_package_url}")
             except AttributeError:
-                support_package_url = self.support_package_url
+                # If the app specifies a support revision, use it;
+                # otherwise, use the support revision named by the template
+                try:
+                    support_revision = app.support_revision
+                except AttributeError:
+                    # No support revision specified; use the template-specified version
+                    try:
+                        support_revision = self.support_revision(app)
+                    except KeyError:
+                        # No template-specified support revision
+                        raise MissingSupportPackage(
+                            python_version_tag=self.python_version_tag,
+                            platform=self.platform,
+                            host_arch=self.host_arch,
+                        )
+
+                support_package_url = self.support_package_url(support_revision)
                 custom_support_package = False
                 self.logger.info(f"Using support package {support_package_url}")
 
             if support_package_url.startswith(("https://", "http://")):
-                try:
-                    self.logger.info(f"... pinned to revision {app.support_revision}")
-                    # If a revision has been specified, add the revision
-                    # as a query argument in the support package URL.
-                    # This is a lot more painful than "add arg to query" should
-                    # be because (a) url splits aren't appendable, and
-                    # (b) Python 3.5 doesn't guarantee dictionary order.
-                    url_parts = list(urlsplit(support_package_url))
-                    query = list(parse_qsl(url_parts[3]))
-                    query.append(("revision", app.support_revision))
```

Comment:
If this URL query argument functionality is being removed, then it should also be removed from `support_revision` in configuration.rst.